pkg/prom/helpers.go

```diff
@@ -11,7 +11,7 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-const PrometheusTroubleshootingURL = "http://docs.kubecost.com/custom-prom#troubleshoot"
+const PrometheusTroubleshootingURL = "https://gitbook.kubecost.com/v/1.0x/install-and-configure/install/custom-prom"
```

Comment:
Could we change this to https://www.opencost.io/docs/integrations/prometheus and we'll improve the docs there rather than refer to Kubecost's documentation?

Reply:
@mattray , I've wrapped up the work on my end! Should there be any changes you deem necessary, just let me know. Otherwise, it's all set for merging if you're satisfied with how things stand.